packages/web-runtime/src/layouts/Loading.vue

```diff
@@ -8,7 +8,7 @@
     <oc-spinner size="xlarge" :aria-label="$gettext('Loading')" />
   </div>
 </template>
-<script>
+<script lang="ts">
 import { mapGetters } from 'vuex'
 
 export default {
```

Comment:
You forgot to wrap the object into `defineComponent()` ;-) At least that's what we usually do when we set `lang="ts"` on a script tag.

Reply:
no idea, why I added it at all - probably played around with something and only partially reverted it